server/routes.go

```diff
@@ -720,9 +720,50 @@ func GetModelInfo(req api.ShowRequest) (*api.ShowResponse, error) {
 	fmt.Fprint(&sb, model.String())
 	resp.Modelfile = sb.String()
 
+	ggmlData, err := getGGMLData(model.ModelPath, req.Verbose)
```

Comment:
I believe you can use `llm.LoadModel`:

```
ggml, err := llm.LoadModel(pending.model.ModelPath)
kv := ggml.KV()
...

Reply:
Ah I see we do some post processing for verbose below – above might still be useful for `getGGMLData`